streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/KTableKTableJoinNode.java

```diff
@@ -220,34 +198,26 @@ public KTableKTableJoinNodeBuilder<K, V1, V2, VR> withJoinOtherStoreNames(final
         }
 
         public KTableKTableJoinNodeBuilder<K, V1, V2, VR> withQueryableStoreName(final String queryableStoreName) {
-            this.queryableStoreName = queryableStoreName;
```

Comment:
Is this intentional?

Reply:
Oops! Good catch, it's no longer used I meant to remove the entire method.